test/functional/testSettings.js

```diff
@@ -0,0 +1,594 @@
+define(function (require) {
+
+  var registerSuite = require('intern!object');
+  var expect = require('intern/dojo/node!expect.js');
+  var ScenarioManager = require('intern/dojo/node!../fixtures/scenarioManager');
+  var pollUntil = require('intern/dojo/node!leadfoot/helpers/pollUntil');
+  var Common = require('../support/pages/Common');
+  var SettingsPage = require('../support/pages/SettingsPage');
+  var HeaderPage = require('../support/pages/HeaderPage');
+  var config = require('intern').config;
+  var url = require('intern/dojo/node!url');
+  var _ = require('intern/dojo/node!lodash');
+
+
+  registerSuite(function () {
+    var common;
+    var settingsPage;
+    var headerPage;
+    var scenarioManager;
+    var remote;
+
+    var expectedAlertText = 'Are you sure you want to remove this index pattern?';
+    return {
+      // on setup, we create an settingsPage instance
+      // that we will use for all the tests
+      setup: function () {
+        common = new Common(this.remote);
+        settingsPage = new SettingsPage(this.remote);
+        headerPage = new HeaderPage(this.remote);
+        scenarioManager = new ScenarioManager(url.format(config.elasticsearch));
+        remote = this.remote;
+      },
+
+      beforeEach: function () {
+
+        // start each test with an empty kibana index
+        return scenarioManager.unload('emptyKibana')
+          .then(function () {
+            return common.sleep(1000);
```

Comment:
`scenarioManager` methods return promises, so in theory, you shouldn't need to wait for things to settle via a timeout here. Were you having issues without the timeouts? If so, that should probably be fixed in `scenarioManager` somehow.